hooks/useCopyToClipboard.ts

```diff
@@ -19,7 +19,7 @@ export const useCopyToClipboard = () => {
 
   useEffect(() => {
     if (copied) {
-      const timerId = setTimeout(() => setCopied(false), 3000);
+      const timerId = setTimeout(() => setCopied(false), 2000);
```

Comment:
Any particular reason for changing in duration ? I believe 3000ms is sufficient enough